### PR TITLE
Add cookie message

### PR DESCRIPTION
**Pull request by Stormheg**
image

### CODE REVIEW

**Comment by Stormheg** [comment]
This assumes a page at `/cookie-policy/` exists.

File: wagtailio/static/css/components/cookie-message.scss [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.cookie {
+    $root: &;
+    padding: 0px 10px 10px 10px;
```
Comment (SimonDEvans) [inline wagtailio/static/css/components/cookie-message.scss:3]: We could simplify this slightly to: `padding: 0 10px 10px;`

File: wagtailio/static/css/components/cookie-message.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    position: fixed;
+    right: auto;
+    bottom: 0;
+    left: auto;
+    z-index: 13;
+    display: none; // Doesn't display unless there isn't a cookie
+    width: 100%;
+    pointer-events: none;
```
Comment (SimonDEvans) [inline wagtailio/static/css/components/cookie-message.scss:11]: I'm not sure there's a need for removing pointer events if the element is already hidden with `display: none;`

File: wagtailio/static/css/components/cookie-message.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    // Applied if cookie just added
+    &.inactive {
+        display: block;
+        transform: translate3d(0, 100%, 0);
```
Comment (SimonDEvans) [inline wagtailio/static/css/components/cookie-message.scss:16]: Very picky, but you could use `transform: translateY(100%);` instead as we aren't concerned with `X` or `Z` here.

File: wagtailio/static/css/components/cookie-message.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    &.active {
+        display: block;
+        transform: translate3d(0, 0, 0);
```
Comment (SimonDEvans) [inline wagtailio/static/css/components/cookie-message.scss:21]: As above.

File: wagtailio/static/css/components/cookie-message.scss [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        &:hover {
+            color: $color--secondary;
+        }
+    }
+
+    &__btn {
+        display: block;
+        width: 24px;
+        height: 24px;
+        pointer-events: auto;
```
Comment (SimonDEvans) [inline wagtailio/static/css/components/cookie-message.scss:58]: Do we need this one?

**Pull request merged**